### PR TITLE
WIP: Add Boilerplate update script

### PR DESCRIPTION
**Pull request by RobLoach**

### CODE REVIEW

File: jst-nuke-boilerplate [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+cwd="`pwd`"
+
+for d in `echo ${root}jstransformer*/`; do
+  repo="`basename $d`"
+  echo "Updating $repo"
+  cd "$d"
+  rm -rf node_modules
+  git pull git@github.com:jstransformers/boilerplate.git master
+  git push origin master
```
Comment (TimothyGu) [inline jst-nuke-boilerplate]: I'd prefer the inclusion of a step that allows manually making sure the merging of boilerplate is correct before pushing the changes.

Comment (RobLoach) [inline jst-nuke-boilerplate]: Agreed, I'd like it to check if there are merge conflicts, and then report them to a bad.txt or something.

File: jst-nuke-boilerplate-push [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+# Pulls Boilerplate into each package and attempts to push the changes up.
```
Comment (ForbesLindesay) [inline jst-nuke-boilerplate-push]: This description seems incorrect

File: jst-nuke-boilerplate-push [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+root=$(awk -F '=' '{if (! ($0 ~ /^;/) && $0 ~ /root/) print $2}' config.ini)
+
+cwd="`pwd`"
+
+for d in `echo ${root}jstransformer*/`; do
+  repo="`basename $d`"
+  echo "Processing $repo"
+  cd "$d"
+  git add -A
+  git commit -m "Update Boilerplate"
```
Comment (ForbesLindesay) [inline jst-nuke-boilerplate-push]: Should the commit message still be "Update Boilerplate"? Should there be something to skip the commit if there are no changes?

**Comment by ForbesLindesay** [comment]
At a glance, this looks awesome. Feel free to merge when you're happy with it, or ping me if you want me to take another look.

File: jst-nuke-boilerplate-push (hunk 3 above)
Comment (RobLoach) [reply jst-nuke-boilerplate-push]: Since `--allow-empty` isn't set, empty commits arn't made.

Comment (ForbesLindesay) [reply jst-nuke-boilerplate-push]: 👍

**Pull request merged**